x-pack/test/security_solution_api_integration/scripts/mki_api_ftr_execution.ts

```diff
@@ -88,23 +89,48 @@ export const cli = () => {
         level: 'info',
         writeTo: process.stdout,
       });
+      const environment = process.env.ENVIRONMENT ? `${process.env.ENVIRONMENT}` : 'qa';
 
       const PROXY_URL = process.env.PROXY_URL ? process.env.PROXY_URL : undefined;
       const PROXY_SECRET = process.env.PROXY_SECRET ? process.env.PROXY_SECRET : undefined;
       const PROXY_CLIENT_ID = process.env.PROXY_CLIENT_ID ? process.env.PROXY_CLIENT_ID : undefined;
-      const API_KEY = process.env.CLOUD_QA_API_KEY ? process.env.CLOUD_QA_API_KEY : undefined;
+
+      // Checking if API key is either provided via env variable or in ~/.elastic.cloud.json
+      // This works for either local executions or fallback in case proxy service is unavailable.
+      if (!process.env.CLOUD_API_KEY && !getApiKeyFromElasticCloudJsonFile(environment)) {
```

Comment:
The same as above regarding double `getApiKeyFromElasticCloudJsonFile(environment)` invocation.